### ISSUE TITLE
Fix Sequitur installation in python

### ISSUE THREAD

**Issue by ronaldtse** [interscript/interscript-ruby#584]
e.g. https://github.com/interscript/interscript/pull/573/checks?check_run_id=1287104884
```
pip3 install setuptools numpy
3
  pip3 install git+https://github.com/sequitur-g2p/sequitur-g2p@9b7c505
4
  shell: /bin/bash -e {0}
5
  env:
6
    pythonLocation: /Users/runner/hostedtoolcache/Python/3.9.0/x64
7
Requirement already satisfied: setuptools in /Users/runner/hostedtoolcache/Python/3.9.0/x64/lib/python3.9/site-packages (49.2.1)
8
Collecting numpy
9
  Downloading numpy-1.19.2.zip (7.3 MB)
10
  Installing build dependencies: started
11
  Installing build dependencies: finished with status 'done'
12
  Getting requirements to build wheel: started
13
  Getting requirements to build wheel: finished with status 'done'
14
    Preparing wheel metadata: started
15
    Preparing wheel metadata: finished with status 'done'
16
Building wheels for collected packages: numpy
17
  Building wheel for numpy (PEP 517): started
18
  Building wheel for numpy (PEP 517): still running...
19
  Building wheel for numpy (PEP 517): finished with status 'done'
20
  Created wheel for numpy: filename=numpy-1.19.2-cp39-cp39-macosx_10_15_x86_64.whl size=4625216 sha256=5535e7004b0fead4260ac8147494380ab1e63079d8521afc67eeb19bdf288e75
21
  Stored in directory: /Users/runner/Library/Caches/pip/wheels/a3/17/dd/f2dba23a35bb6008732772ccfb13d3d0e537fbc6919ce6862b
22
Successfully built numpy
23
Installing collected packages: numpy
24
Successfully installed numpy-1.19.2
25
Collecting git+https://github.com/sequitur-g2p/sequitur-g2p@9b7c505
26
  Cloning https://github.com/sequitur-g2p/sequitur-g2p (to revision 9b7c505) to /private/var/folders/24/8k48jl6d249_n_qfxwsl6xvm0000gn/T/pip-req-build-vv68wy6o
27
  WARNING: Did not find branch or tag '9b7c505', assuming revision or ref.
28
  Installing build dependencies: started
29
  Installing build dependencies: finished with status 'done'
30
  Getting requirements to build wheel: started
31
  Getting requirements to build wheel: finished with status 'error'
32
  ERROR: Command errored out with exit status 1:
33
   command: /Users/runner/hostedtoolcache/Python/3.9.0/x64/bin/python /Users/runner/hostedtoolcache/Python/3.9.0/x64/lib/python3.9/site-packages/pip/_vendor/pep517/_in_process.py get_requires_for_build_wheel /var/folders/24/8k48jl6d249_n_qfxwsl6xvm0000gn/T/tmpujyv3ag1
34
       cwd: /private/var/folders/24/8k48jl6d249_n_qfxwsl6xvm0000gn/T/pip-req-build-vv68wy6o
35
  Complete output (23 lines):
36
  init_dgelsd failed init
37
  Traceback (most recent call last):
38
    File "/Users/runner/hostedtoolcache/Python/3.9.0/x64/lib/python3.9/site-packages/pip/_vendor/pep517/_in_process.py", line 280, in <module>
39
      main()
40
    File "/Users/runner/hostedtoolcache/Python/3.9.0/x64/lib/python3.9/site-packages/pip/_vendor/pep517/_in_process.py", line 263, in main
41
      json_out['return_val'] = hook(**hook_input['kwargs'])
42
    File "/Users/runner/hostedtoolcache/Python/3.9.0/x64/lib/python3.9/site-packages/pip/_vendor/pep517/_in_process.py", line 114, in get_requires_for_build_wheel
43
      return hook(config_settings)
44
    File "/private/var/folders/24/8k48jl6d249_n_qfxwsl6xvm0000gn/T/pip-build-env-r6xoctrr/overlay/lib/python3.9/site-packages/setuptools/build_meta.py", line 149, in get_requires_for_build_wheel
45
      return self._get_build_requires(
46
    File "/private/var/folders/24/8k48jl6d249_n_qfxwsl6xvm0000gn/T/pip-build-env-r6xoctrr/overlay/lib/python3.9/site-packages/setuptools/build_meta.py", line 130, in _get_build_requires
47
      self.run_setup()
48
    File "/private/var/folders/24/8k48jl6d249_n_qfxwsl6xvm0000gn/T/pip-build-env-r6xoctrr/overlay/lib/python3.9/site-packages/setuptools/build_meta.py", line 253, in run_setup
49
      super(_BuildMetaLegacyBackend,
50
    File "/private/var/folders/24/8k48jl6d249_n_qfxwsl6xvm0000gn/T/pip-build-env-r6xoctrr/overlay/lib/python3.9/site-packages/setuptools/build_meta.py", line 145, in run_setup
51
      exec(compile(code, __file__, 'exec'), locals())
52
    File "setup.py", line 29, in <module>
53
      import numpy
54
    File "/private/var/folders/24/8k48jl6d249_n_qfxwsl6xvm0000gn/T/pip-build-env-r6xoctrr/overlay/lib/python3.9/site-packages/numpy/__init__.py", line 286, in <module>
55
      raise RuntimeError(msg)
56
  RuntimeError: Polyfit sanity test emitted a warning, most likely due to using a buggy Accelerate backend. If you compiled yourself, see site.cfg.example for information. Otherwise report this to the vendor that provided NumPy.
57
  RankWarning: Polyfit may be poorly conditioned
58
  
59
  ----------------------------------------
60
ERROR: Command errored out with exit status 1: /Users/runner/hostedtoolcache/Python/3.9.0/x64/bin/python /Users/runner/hostedtoolcache/Python/3.9.0/x64/lib/python3.9/site-packages/pip/_vendor/pep517/_in_process.py get_requires_for_build_wheel /var/folders/24/8k48jl6d249_n_qfxwsl6xvm0000gn/T/tmpujyv3ag1 Check the logs for full command output.
61
Error: Process completed with exit code 1.
```

**Comment by ronaldtse** [comment]
@CAMOBAP would you have time to take a look? Thanks!

**Comment by webdev778** [comment]
This was happened due to the new version of Python 3.9 has been released 10.5.

**Comment by webdev778** [comment]
PR #585
